master/reference/public-cloud/aws.md

```diff
@@ -49,74 +49,28 @@ Keep reading for information on when IPIP is required in AWS.
 You can check if your hosts have successfully established BGP sessions with
 one another using `calicoctl node status`.
 
-#### Allow Traffic Within a Single VPC Subnet
+#### Allow Container Traffic
 
-By default, each EC2 instance performs source/destination checks on inbound
-and outbound traffic. This ensures that the instance is the source or
-destination of any traffic it sends or receives.
-
-Since Calico allocates a unique IP address to each container that isn't the IP
-of its host, and since Calico doesn't encapsulate container traffic by default,
-traffic to/from containers will be dropped by the AWS src/dst checks
+By default, AWS drops container traffic with an unrecognized IP.
```

Comment:
I think the key point for each of these sections is not so much removing a lot of the content, but rather we should be focusing on what the user needs to do as opposed to why they need to do it.

For example, this section starts by explaining why users need to disable src/dst checks at a fairly technical level.  For a lot of users they either don't care or don't understand, and will get turned off.

How do you feel about this sort of wording - basically switching the order so the "what you need to do" comes first and then "oh and if you care about why, well here's why"

```markdown
## Within a Single VPC Subnet

To use Calico within a single VPC subnet you will need to _disable_ AWS src/dst 
checks on each EC2 instance in your cluster [as described in the AWS documentation](link/to/docs).

When src/dst checking is enabled, the AWS fabric enforces reverse path filtering which 
drops traffic between containers in the cluster after it leaves the host EC2 instance.
```